src/crypto/rx-slow-hash.c

```diff
@@ -236,7 +274,7 @@ void rx_slow_hash(const uint64_t mainheight, const uint64_t seedheight, const ch
   char *hash, int miners, int is_alt) {
   uint64_t s_height = rx_seedheight(mainheight);
   int toggle = (s_height & SEEDHASH_EPOCH_BLOCKS) != 0;
-  randomx_flags flags = RANDOMX_FLAG_DEFAULT;
+  randomx_flags flags = enabled_flags() & ~disabled_flags();
```

Comment:
We can delete cpuid() and check_aes_hw() now, nothing calls them. (Lines 103-131.)

Reply:
I have removed the unused functions.